content-src/experiments/no-more-404s.yaml

```diff
@@ -92,74 +92,51 @@ order: 6
-
   <p>We launched the No More 404s experiment with a few goals in mind. We know from user research that 404 pages are a persistent nuisance on the Web, and a No More 404s add-on seemed like a really sweet utility to help steer users toward their intended content. Additionally, the Test Pilot team wanted to find out how a partner organization like the Internet Archive could help bring testable features into Firefox.</p>
-
   <p><strong>Here’s how it went down</strong></p>
-
   <p>When we launched No More 404s, the user interface was simple and unobtrusive. If a participant visited a 404 on the Web – specifically if a site that returned a HTTP header code 404 – the No More 404s experiment would send a request to the Wayback Machine to see if the service had an archived version of the page. If one existed, No More 404s would drop a notification bar on the page with a link to it. Users could then visit the content archived in the Wayback Machine, dismiss the notification bar, or just continue browsing as usual.</p>
-
   <figure>
   <img src="/static/images/experiments/no-more-404s/experiments_experimentdetail/2/4/24ddd4335aca6b96cca9106a6f3411d2_image_1470245154_0440.jpg">
   <figcaption>We launched No More 404s with a restrained design.</figcaption>
   </figure>
-
   <p>Later on in the experiment we made a few changes to the add-on. First, we added in more common error codes associated with missing Web assets. Then we made a much larger change when we asked our design team to increase the visibility of the add-on. We wanted to see how these changes would affect user perception of the experiment.</p>
-
   <figure>
   <img src="/static/images/experiments/no-more-404s/graduation/design-update.png" width="100%" height="auto" />
   <figcaption>In December we changed to something a bit crazier.</figcaption>
   </figure>
-
   <p><strong>Here's what we learned</strong></p>
-
   <p>Both teams wanted to get a sense of how many of the URLs that display 404 errors today had pages archived in the Wayback Machine. <a href="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/edit#gid=287298530" target="_blank">We tracked total 404s</a> encountered by Test Pilot participants (1,275,165) as well as results returned by the Archive (185,081). Over the course of the experiment, we averaged a result 14.5 percent of the time, but also observed that over the course of the experiment the daily average rose.</p>
-
   <figure>
-  <iframe class="dashboard-embed" src="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/pubchart?oid=1805424694&amp;format=interactive" width=”600” height=”380”></iframe>
+  <iframe class="dashboard-embed" src="https://docs.google.com/spreadsheets/d/1p2cZFy5DiM0JAFhp4lIisLtTeu4Y3fIusQLE9obOx5s/pubchart?oid=1805424694&amp;format=interactive"</iframe>
```

Comment:
This looks bad. I think we're missing a closing ">" before the closing `</iframe>`.